mlir/test/Dialect/Transform/test-pass-application.mlir

```diff
@@ -111,9 +111,9 @@ module attributes {transform.with_named_sequence} {
   transform.named_sequence @__transform_main(%arg1: !transform.any_op) {
     %1 = transform.structured.match ops{["func.func"]} in %arg1 : (!transform.any_op) -> !transform.any_op
 
-    // func-bufferize can be applied only to ModuleOps.
+    // duplicate-function-elimination can be applied only to ModuleOps.
     // expected-error @below {{pass pipeline failed}}
-    transform.apply_registered_pass "func-bufferize" to %1 : (!transform.any_op) -> !transform.any_op
+    transform.apply_registered_pass "duplicate-function-elimination" to %1 : (!transform.any_op) -> !transform.any_op
```

Comment:
AFAIU here 'func-bufferize' just happened to be used. It is not really playing any specific role. right?

Reply:
That's right. This test tries to apply a ModulePass to a function and checks if an error is produced. Any ModulePass will do for the test.